src/config/fonts/styles.css

```diff
@@ -1,96 +1,6 @@
 @font-face {
-  font-family: 'Arizona Serif';
-  src: url('./ABCArizonaSerif-Light.ttf') format('truetype');
+  font-family: 'ArizonaSerif';
+  src: url('./ArizonaSerif/ArizonaSerif-Light.otf') format('truetype');
+  font-weight: 200;
   font-style: normal;
 }
-
-@font-face {
-  font-family: 'Arizona Serif';
-  src: url('./ArizonaSerif-Light.woff') format('woff');
-  font-style: normal;
-}
-
-@font-face {
-  font-family: 'Arizona Serif';
-  src: url('./ArizonaSerif-Light.woff2') format('woff2');
-  font-style: normal;
-}
-
```

Comment:
Why did you change the font setup? This doesn't seem to work, e.g. the buttons have the MUI default font Roboto now. I can also try to fix this in my theme branch once this is merged, if there was an issue with the MUI theme.

Reply:
Done